src/test/java/org/jabref/logic/layout/format/HTMLCharsTest.java

```diff
@@ -27,16 +27,15 @@ private static Stream<Arguments> provideBasicFormattingData() {
                 Arguments.of("hallo", "hallo"),
                 Arguments.of("Réflexions sur le timing de la quantité", "Réflexions sur le timing de la quantité"),
                 Arguments.of("%%%", "\\%\\%\\%"),
-                Arguments.of("People remember 10%, 20%…Oh Really?", "{{People remember 10\\%, 20\\%…Oh Really?}}"),
                 Arguments.of("h&aacute;llo", "h\\'allo"),
                 Arguments.of("&imath; &imath;", "\\i \\i"),
                 Arguments.of("&imath;", "\\i"),
                 Arguments.of("&imath;", "\\{i}"),
                 Arguments.of("&imath;&imath;", "\\i\\i"),
-                Arguments.of("&auml;", "{\\\"{a}}"),
-                Arguments.of("&auml;", "{\\\"a}"),
+                Arguments.of("&auml;", "\\\"{a}"),
```

Comment:
The old cases should be kept. Add the braces to the expected output.

Reason: modifying tests is OK. However, existing inputs should be kept - and not just removed. (If they are removed, they should be removed with reason!)

Thus, re-add the old test cases.